apps/wolfsshd/configuration.c

```diff
@@ -474,10 +475,11 @@ static int HandleInclude(WOLFSSHD_CONFIG *conf, const char *value)
 
     /* Use wildcard */
     if (found) {
-#ifdef __unix__
+#if defined(__unix__) || defined(__unix) || \
+    (defined(__APPLE__) && defined(__MACH__))
         int ret;
         struct dirent *dir;
-        DIR *d;
+        WDIR d;
         char *path;
         char *filepath = (char*)WMALLOC(PATH_MAX, NULL, 0);
```

Comment:
We need to check the return of this `WMALLOC`.

Reply:
A little later, the path variable is also malloced without a check, and isn't freed. Fixing.